tests/Doctrine/Tests/ORM/Functional/VersionedOneToOneTest.php

```diff
@@ -50,8 +50,11 @@ public function testSetVersionOnCreate()
         $this->_em->persist($firstRelatedEntity);
         $this->_em->flush();
 
+        $firstRelatedEntity->name = 'Fred2';
+        $this->_em->flush();
+
         $firstEntity = $this->_em->getRepository('Doctrine\Tests\Models\VersionedOneToOne\FirstRelatedEntity')
-            ->findOneBy(array('name' => 'Fred'));
+            ->findOneBy(array('name' => 'Fred2'));
```

Comment:
Can you please add a test method, instead of reusing an existing one for displaying the issue?